test/integration/fixtures/blocks/core__comments__deprecated-1.serialized.html

```diff
@@ -0,0 +1,29 @@
+<!-- wp:comments {"className":"comments-post-extra has-luminous-vivid-orange-color has-text-color","textColor":"luminous-vivid-orange"} -->
```

Comment:
Compare this line to the [original input](https://github.com/WordPress/gutenberg/pull/40506/files#diff-556db1e883beda2196365535b3d8872222aa469a91a3b67bca6e7a030c3fdd3eR1).

The block-supports generated classes shouldn't be here, i.e. we'd like the following instead:

The block-supports generated classes shouldn't be here, i.e. we'd like the following instead:

```suggestion
<!-- wp:comments {"textColor":"luminous-vivid-orange","className":"comments-post-extra"} -->
```

We'll want to change the deprecation accordingly.

See https://github.com/WordPress/gutenberg/pull/40506#issuecomment-1174074568 for more context.